src/views/studio/CreateEditChannelView/CreateEditChannelView.styles.ts

```diff
@@ -38,7 +40,7 @@ export const TitleContainer = styled.div`
 export const InnerFormContainer = styled.div`
   width: 100%;
   margin-top: 50px;
-  padding-bottom: 250px;
+  padding-bottom: ${VIDEO_WORKSPACE_TABS_BAR_HEIGHT}px;
```

Comment:
Why?

Reply:
I need some additional space for ActionBar. Without this padding I can't see <select> element fully, because it's hidden:
![image](https://user-images.githubusercontent.com/51168865/144573334-6942dc55-f450-4f29-8390-484f8a1a5618.png)

With: 
![image](https://user-images.githubusercontent.com/51168865/144573443-14f38932-5c54-4e9a-a02c-155d6be3fc01.png)

Reply:
What does it have to do with the video workspace though? It seems completely unrelated to me

Reply:
I don't know why I picked this, for some reason I thought that this variable contains the height of the action bar 😅 
Anyway, should be good now. Basically did the same as we have in the VideoWorkspace form.